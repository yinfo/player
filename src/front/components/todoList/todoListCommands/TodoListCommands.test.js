// @flow

import React from 'react';
import { shallow } from 'enzyme';
import TodoListCommands from './TodoListCommands';

describe('TodoListCommands component', () => {
  it('renders as expected', () => {
    const component = shallow(
      <TodoListCommands>
        <p>a child</p>
      </TodoListCommands>,
    );
    expect(component).toMatchSnapshot();
  });
});
