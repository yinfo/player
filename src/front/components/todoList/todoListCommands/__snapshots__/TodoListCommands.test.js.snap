// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoListCommands component renders as expected 1`] = `ShallowWrapper {}`;
